frontend/src/views/Home.vue

```diff
@@ -11,7 +21,9 @@ import QuizCard from '@/components/QuizCard.vue';
         </div>
         <!-- Quiz details will be inserted as attributes to QuizCard components
             for each quiz in quizzes list -->
-        <QuizCard v-for="num in 7" v-bind:key="num" />
+        <div v-for="num in 7" :key="num">
```

Comment:
Do you prefer `:key` to `v-bind:key` ?  If yes, why ?

Reply:
It's shorter 🤣

Reply:
For sure, but not uniform
I mean `v-for`, `v-bind`, `v-if`... looks better than `v-for`, `:`, `v-if`, ...😂

Reply:
Yep but it means that across all the project, you want to use `v-*`.. And you don't want to use the shorthand.. Never... If so, then why not.

Feel free to change it in a follow up PR